src/lzma_worker.js

```diff
@@ -2628,7 +2628,8 @@ var LZMA = (function () {
     return {
         /** xs */
         compress:   compress,
-        decompress: decompress
+        decompress: decompress,
+        terminateWorker: function() {}
```

Comment:
I’ve done some thinking about this, and maybe it would be even better to have some way of accessing the web worker object itself (e.g. `LZMA().worker()`, which would return `null`/undefined/… in case there is no worker); Then, users could call `.terminate()` themselves. This might be a little more intuitive and would have the advantage of providing access to other parts of the Web Worker API (and especially future additions to it).